birdhouse/deployment/triggerdeploy.sh

```diff
@@ -93,17 +106,46 @@ should_trigger() {
         AHEAD_CNT="`git rev-list --left-only --count ${local}...${remote}`"
         echo "BEHIND_CNT=$BEHIND_CNT" >> $TMP_SAVE_FILE
         echo "AHEAD_CNT=$AHEAD_CNT" >> $TMP_SAVE_FILE
+        echo "local=$local" >> $TMP_SAVE_FILE
```

Comment:
Why does `local` and `remote` doesn't use the same case as `BEHIND_CNT` and `AHEAD_CNT`?

Reply:
Oh yeah, that's because they are variables inside the while loop that do not survive outside the loop since data is feed in by a pipe.  It's a new shell for each loop, see https://github.com/bird-house/birdhouse-deploy/blob/4d9b1ae6009a79bf0a3e0a135b3f8420d3d8cc65/birdhouse/deployment/triggerdeploy.sh#L99-L100

I usually use lowercase for transient variables like that.  They actually survive simply because I write it out to a tmp file and read it back in at the end of the loop https://github.com/bird-house/birdhouse-deploy/blob/4d9b1ae6009a79bf0a3e0a135b3f8420d3d8cc65/birdhouse/deployment/triggerdeploy.sh#L109-L114